src/js/components/Input/Input.js

```diff
@@ -0,0 +1,45 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { getClassName } from '../../utils/helpers'
+
+class Input extends PureComponent {
```

Comment:
Should this be `InputGroup` instead.. since it's outputting that component? 
http://localhost:3002/patterns/components/input-group.html

And `Input` would just be the base `.Input`? 
http://localhost:3002/patterns/components/Input.html

Reply:
I went back and forth on if I should split `Input` from `InputGroup`.  Through my testing, I saw no impact on always allowing the div wrapper of InputGroup.  So I thought this might simplify the approach when trying to add the appropriate icon, instead of trying to remember when.  I'm open to a change here as well.